openapi.yaml

```diff
@@ -12,106 +12,458 @@ info:
         '201':
-          description: Pet successfully created
+          description: Member successfully created
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/PetResult'
+                $ref: '#/components/schemas/MemberResult'
         '400':
           $ref: '#/components/responses/400'
         '409':
           $ref: '#/components/responses/409'
         '500':
           $ref: '#/components/responses/500'
-  /pets/{id}:
+  /members/{id}:
+    get:
+      summary: Retrieve details of a specific member.
+      tags:
+        - members
+      description: Get Member by unique identifier.
+      operationId: getMemberById
+      parameters:
+        - $ref: '#/components/parameters/memberId'
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MemberResult'
+        '404':
+          $ref: '#/components/responses/404'
+        '500':
+          $ref: '#/components/responses/500'
+    patch:
+      summary: Update details of a specific member.
+      tags:
+        - members
+      description: Update a member by unique identifier.
+      operationId: updateMemberById
+      parameters:
+        - $ref: '#/components/parameters/memberId'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/MemberPatchBody'
+      responses:
+        '200':
+          description: Details of the updated member.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MemberResult'
+        '404':
+          $ref: '#/components/responses/404'
+        '500':
+          $ref: '#/components/responses/500'
+    delete:
+      summary: Remove a specific member of the library.
+      tags:
+        - members
+      description: Remove a member by unique identifier.
+      operationId: deleteMemberById
+      parameters:
+        - $ref: '#/components/parameters/memberId'
+      responses:
+        '204':
+          description: Member successfully deleted.
+        '404':
+          $ref: '#/components/responses/404'
+        '500':
+          $ref: '#/components/responses/500'
+  /borrows:
     get:
-      summary: Info for a specific pet
+      summary: Retrieve a list of all borrowed books from the library.
       tags:
-        - pets
-      description: Get pet by unique ID
-      operationId: getPetById
+        - borrows
+      description: Get all Borrowed Books
+      operationId: getBorrows
       parameters:
-        - name: id
-          in: path
-          description: Unique Pet ID
-          required: true
+        - $ref: '#/components/parameters/pageNumber'
+        - $ref: '#/components/parameters/pageSize'
+        - in: query
+          name: filter[book_id]
```

Comment:
camelCase. Make sure all of your code follows the same naming convention.

Reply:
true, fixed.